integrations-and-supported-tools/catalyst/notebooks/Neptune_Catalyst.ipynb

```diff
@@ -55,7 +55,7 @@
    },
    "outputs": [],
    "source": [
-    "! pip install -U catalyst ipywidgets neptune-client"
+    "! pip install -U catalyst ipywidgets neptune-client tensorboard torchvision"
```

Comment:
Why include tensorboard?

Reply:
its a catalyst requirement
https://github.com/neptune-ai/examples/actions/runs/3851466245/jobs/6562673946

Reply:
Got it!